doc/modules/cross_validation.rst

```diff
@@ -300,6 +300,23 @@ section.
     * :ref:`sphx_glr_auto_examples_model_selection_plot_cv_predict.py`,
     * :ref:`sphx_glr_auto_examples_model_selection_plot_nested_cross_validation_iris.py`.
 
+
+.. _cv_significance_evaluation:
+
+Cross-validation significance evaluation
+----------------------------------------
+
+Significance of cross validation scores can be evaluated using the
+:func:`permutation_test_score` function. The function returns a p-value, which
+approximates the probability that the average cross-validation score would be
+obtained by chance if the target is independent of the data.
+
+It also returns cross validation scores for each permutation of y labels. It
```

Comment:
Talking about return value here confuses the matter if what we want to talk about is how the P value is constructed

Reply:
This documentation is for user guide. The function description and the paper cited give more details about how the p-value is constructed.

Reply:
Hi, we're trying to get all of the PR's from the WiMLDS sprint wrapped up, are we waiting on 
@aditi9783, or approval from another reviewer? Thanks

Not completely sure what's going on with this one @reshamas

Reply:
Hi @thomasjpfan, @reshamas asked me to take over this PR, but I see that Nicolas Hug already approved the changes, and it is just waiting for one more reviewer. 

Is the PR good? Or should I move most of the docstring into the user guide?